jenkins/internal-ci/centos-7.9.2009/pr/component_test/pyutils-premerge.groovy

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env groovy
+// CLEANUP REQUIRED
+pipeline { 
+    agent {
+        node {
+            label "docker-${OS_VERSION}-node"
+        }
+    }
+
+    options { 
+        skipDefaultCheckout()
+        timeout(time: 180, unit: 'MINUTES')
+        timestamps()
+        ansiColor('xterm')  
+    }
+
+
+    parameters {  
+	    string(name: 'CORTX_RE_URL', defaultValue: 'https://github.com/Seagate/cortx-re', description: 'Repo for cortx-re')
+        string(name: 'CORTX_RE_BRANCH', defaultValue: 'main', description: 'Branch for cortx-re')
+        string(name: 'PY_UTILS_URL', defaultValue: 'https://github.com/Seagate/cortx-utils', description: 'Repo for Py-Utils Agent')
+        string(name: 'PY_UTILS_BRANCH', defaultValue: 'main', description: 'Branch for Py-Utils Agent')     
+        // string(name: 'HOST', defaultValue: '-', description: 'Host=FQDN,user=USERNAME,pass=PASSWORD',  trim: true)
+        string(name: 'COMPONENTS_BRANCH', defaultValue: 'main', description: 'Branch for other components',  trim: true)
+        choice(
+            name: 'OS_VERSION',
+            choices: ['centos-7.9.2009', 'centos-7.8.2003'],
+            description: 'OS version of HOST'
+        )
+        choice(
+            name: 'DEPLOY_BUILD_ON_NODES',
+            choices: ["Both", "1node", "3node" ],
```

Comment:
Please check with the Utils team if they need both 1N and 3N. AFIK we can use any one of them.

Reply:
checked with utils 1N is fine for now.